samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1886,6 +1904,17 @@ mfxStatus CDecodingPipeline::RunDecoding()
                 pBitstream = NULL;
                 sts = MFX_ERR_NONE;
                 continue;
+            } else if (MFX_ERR_REALLOC_SURFACE == sts) {
+                mfxVideoParam param;
```

Comment:
this needs to be initialized, add {} like this:
`mfxVideoParam param{};`

Reply:
Sure. I have added it. Thanks.